examples/examplepb/a_bit_of_everything.proto

```diff
@@ -8,10 +8,43 @@ import "google/protobuf/duration.proto";
 import "examples/sub/message.proto";
 import "examples/sub2/message.proto";
 import "google/protobuf/timestamp.proto";
+import "protoc-gen-swagger/options/annotations.proto";
+
+option (grpc_gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
```

Comment:
Don't you need to import the relevant proto file to access options?

Reply:
Ah, that's annotations.proto, I see. I was confused by the package here, importing from `protoc-gen-swagger` and then accessing a `grpc_gateway` namespace. Might just be me but feels like a disconnect?

Reply:
Sounds good to me :)